torch/profiler/profiler.py

```diff
@@ -242,11 +242,11 @@ def export_chrome_trace(self, path: str):
         """
         assert self.profiler
         if path.endswith(".gz"):
-            fp = tempfile.NamedTemporaryFile("w+t", suffix=".json", delete=False)
+            fp = tempfile.NamedTemporaryFile("w+b", suffix=".json", delete=False)
             fp.close()
             retvalue = self.profiler.export_chrome_trace(fp.name)
-            with open(fp.name) as fin:
-                with gzip.open(path, "wt") as fout:
+            with open(fp.name, 'rb') as fin:
```

Comment:
nit: can you use double quotes here since we use it for the surrounding lines?

Reply:
> nit: can you use double quotes here since we use it for the surrounding lines?

Of course! Thank you for your reply.